packages/vscode-graphql-syntax/tests/__snapshots__/markdown-grammar.spec.ts.snap

```diff
@@ -107,6 +107,50 @@ graphql\`                                             |
 const Component = () => {                            | 
   return <div> </div>;                               | 
 };                                                   | 
+\`\`\`                                                  | 
+                                                     | 
+\`\`\`js                                                | 
+const variable = 1;                                  | 
+                                                     | 
```

Comment:
huh, these aren't reporting any graphql tokens here in the spec file...

Reply:
wait... i wonder if our changes in a previous PR mean they won't show up as tokens here anymore. I wonder if there is even a point to this spec anymore. I will need to check. obviously it's already regressed for the rest of the file

Reply:
I will check it out manually as well, but my guess is that it's working with markdown, but with the new markdown injection strategy we were granted by some user with incredible TM knowledge in a vscode issue the team had trouble answering, I don't think our serializer picks up on it's tokenization correctly, because you problably noticed this outlandish injection selector:

```
  "injectionSelector": "L:(meta.embedded.block.javascript | meta.embedded.block.typescript | source.js | source.ts | source.tsx | source.vue | source.svelte | source.astro) -source.graphql -inline.graphql -string -comment",
```

i think now it can universally applying to every embedded js/ts block it's pointed at including... html <script tags? 
i think i would just need to add that here:

```
 "injectTo": [
          "source.js",
          "source.ts",
          "source.js.jsx",
          "source.tsx",
          "source.vue",
          "source.svelte",
          "source.astro",
          "text.html.markdown"
          /// other text.html values??
        ],
        ```

Reply:
I wonder if at this point, we are testing vscode's `text.html.markdown` injector then? I think the contributor said as much, that the markdown fixture and spec is not longer necessary, except for ```graphql ``` tags

Reply:
I'm not sure I'm 100% following on what needs to be done 🤔
For what it's worth, I did open the fixtures manually and they seem fine

But yea, while I don't understand the test setup 100%, and it seems that they don't necessarily include any syntax definitions except for the ones defined in this repo, this may need to be fixed up in the future! Not sure if it's that pressing though?

<img width="422" alt="Screenshot 2024-03-05 at 14 21 20" src="https://github.com/graphql/graphiql/assets/2041385/51da3263-8185-40b9-b391-2ecd5e7a46e5">

Reply:
hey sorry, nothing needs to be done here, I am musing on another issue unrelated that I just noticed while reviewing this PR, I digress!

but if you _want_, you can remove the update to the markdown snap/fixture, because I'm about to remove them!